api/src/test/java/org/openmrs/hl7/HL7ServiceTest.java

```diff
@@ -90,6 +92,7 @@ public void processHL7InQueue_shouldCreateHL7InArchiveAfterSuccessfulParsing() t
 		if (tempDir.exists() && tempDir.isDirectory())
 			assertTrue(deleteDirectory(tempDir));
 		
+			assertEquals(true, OpenmrsUtil.deleteDirectory(tempDir));
```

Comment:
this line needs to be removed and the import of `OpenmrsUtil`

Reply:
removed but this OpenmrsUtil import is used in several places